cloudstate/cloudstate.py

```diff
@@ -1,3 +1,8 @@
+"""
+Copyright 2020 Lightbend Inc.
```

Comment:
The date is correct? Wouldn't it be 2019?

Reply:
2019 as the year of the very first creation of the file or its last update? what's more appropriate?